NamingSystem/no-basis-icpc-2.namingsystem.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<NamingSystem xmlns="http://hl7.org/fhir" >
+    <id value="no-basis-icpc-2"/>
+    <meta>
+        <versionId value="2.1.1" />
+    </meta>
+    <name value="ICPC-2 (International Classification of Primary Care) (PH3C)" />
+    <status value="active" />
+    <kind value="codesystem" />
+    <date value="2021-11-02"/>
+    <publisher value="HL7 Norway" />
+    <responsible value="Wonca International Classification Committee (WICC)"/>
+    <description value="In Norway primary care uses ICPC-2 to document contact-reason, health related problem and diagnosis." />
+    <uniqueId>
+        <type value="uri" />
+        <value value="http://hl7.org/fhir/sid/icpc-2" />
+        <preferred value="true" />
+    </uniqueId>
+    <uniqueId>
+        <type value="oid" />
+        <value value="2.16.840.1.113883.6.139" />
+        <preferred value="false" />
+    </uniqueId>
+    <uniqueId>
+        <type value="oid" />
+        <value value="2.16.578.1.12.4.1.1.7170" />
```

Comment:
Endre til: http://hl7.org/fhir/sid/icpc-2 :)

Reply:
Du mener i profilen, ikke i namingsystem?

Reply:
Jeg har oppdatert profilen, og endret samtidig navnet på NamingSystem siden den ikke gikk gjennom java validatoren (warning på navnet fra det internasjonale navnesystemet).

Reply:
Ja, jeg mente profilen. Trodde jeg var på riktig sted. Kikket både tre og fire ganger for å sørge for at jeg var i riktig fil og på riktig sted i filen, men åpenbart ikke godt nok:)